src/FSharpVSPowerTools.Logic/Navigation/GoToDefinitionFilter.fs

```diff
@@ -33,6 +28,12 @@ type internal UrlChangeEventArgs(url: string) =
     inherit EventArgs()
     member __.Url = url
 
+[<NoComparisonAttribute; NoEquality>]
+type GoToDefinitionResolution =
```

Comment:
I'm considering `FoundInternal`, `FoundExternal`, `FoundLoadDirective` and `NotFound`, what do you think?

Those will migrate to Core ultimately and will be consumed by client code, while LoadDirective normally won't.